UnityProject/Assets/Scripts/Core/Sprite Handler/SpriteHandlerManager.cs

```diff
@@ -34,6 +34,12 @@ public static SpriteHandlerManager Instance
 
 	public static void RegisterHandler(NetworkIdentity networkIdentity, SpriteHandler spriteHandler)
 	{
+		if (spriteHandler == null)
+		{
+			Logger.LogError(" Sprite handler was null", Category.SpriteHandler);
```

Comment:
What about this? So we know what object needs fixing.
```suggestion
			Logger.LogError("SpriteHandler was null on {this}.", Category.SpriteHandler);
```

Reply:
Its on the manager so this would grab the manager not the object. But i dont think i need to add the check anyway, as i fixed the issue, so gonna revert it.

Reply:
Oh. Apologies.